cmd/cmd_controller.js

```diff
@@ -58,19 +58,19 @@ class EmbarkController {
 
     const webServerConfig = {};
 
-    if (options.runWebserver != null) {
+    if (options.runWebserver !== null) {
```

Comment:
Seems okay, just keep in mind that if `EmbarkController` is constructor manually in some context, the `!==` will have the effect that an `undefined` option won't be the same as having that option set to `null`. The same goes for the other `!=` to `!==` changes.

Reply:
I'd say in that case, `EmbarkController` should initialize that option with `null` as default, so that in practice it's never `undefined` (unless explicitly set so)